src/Workspaces/Core/MSBuild/MSBuild/MSBuildProjectLoader.Worker_ResolveReferences.cs

```diff
@@ -349,8 +346,15 @@ private async Task<bool> VerifyProjectOutputExistsAsync(string projectPath, Reso
 
                 foreach (var projectFileInfo in projectFileInfos)
                 {
+                    if (projectFileInfo.IsEmpty)
+                    {
+                        continue;
+                    }
```

Comment:
Does this make any difference?

Reply:
if the file info is empty then we expect all path information (`OutputFilePath` and `OutputRefFilePath`) to be `null`